src/components/cc-global-header.vue

```diff
@@ -34,11 +34,14 @@
         :class="{ ['navbar-menu']: true, ['is-active']: isBurgerMenuActive }"
       >
         <div class="navbar-end">
-          <p v-if="errorMessage" class="navbar-item navbar-link is-arrowless">
-            We are having trouble loading the menu
-            <br />
-            Error Message: {{ errorMessage }}.
+          <p v-if="menuLoading" class="navbar-item navbar-link is-arrowless">
+            Loading menu.
           </p>
+          <p
+            v-else
+            v-html="errorMessage"
+            class="navbar-item navbar-link is-arrowless"
+          ></p>
```

Comment:
The previous approach seemed reasonable and would keep HTML contained in the `template`
```suggestion
          <p v-if="errorMessage" class="navbar-item navbar-link is-arrowless">
            We are having trouble loading the menu
            <br />
            Error Message: {{ errorMessage }}.
          ></p>
```

Reply:
We can use `v-else` or `v-if` here. The main idea is to keep HTML in the `template`.

Reply:
Yeah keeping HTML in the template is reasonable. 
I was not very sure if I understood how many messages we were to display, as this one says `We are having trouble loading the menu. Error Message: ` and then prints the error message` 
It seems we have another message to print `Menu Unavailable` which I suppose is to be displayed if all retries fail. 

So I moved it to the JavaScript so e can just have one variable `errorMessage` and then use it to display any number of messages we may want, "menu unavailable` etc instead of having to declare a new variable. 

Personally I prefer leaving it in the template tag and just display the first message with the error.

Reply:
Yeah, I can see your idea that the template might start to get pretty lengthy, notably if we added more conditional cases.

In any case, let's keep the code consistent and conventional by keeping all declarative markup and UI texts in HTML, with only some text coming into the template context via JS. 

I realize there is some magic that Vue is doing underneath things, but from the developer standpoint, we are writing "separate" HTML and JavaScript :smile:

Reply:
I agree too. Let me commit the suggestions